src/data/exercises.json

```diff
@@ -0,0 +1,455 @@
+{
+    "total_count": 10,
```

Comment:
These URL's might need to change slightly once exercises is public (we don't want exercises from random people, only trusted authors or exercises in certain lists), but for now they should work :)